application/F3DOptionsParser.cxx

```diff
@@ -331,6 +331,7 @@ void ConfigurationOptions::GetOptions(F3DAppOptions& appOptions, f3d::options& o
     this->DeclareOption(grp2, "roughness", "", "Roughness coefficient (0.0-1.0)", options.getAsDoubleRef("model.material.roughness"), HasDefault::YES, MayHaveConfig::YES, "<roughness>");
     this->DeclareOption(grp2, "metallic", "", "Metallic coefficient (0.0-1.0)", options.getAsDoubleRef("model.material.metallic"), HasDefault::YES, MayHaveConfig::YES, "<metallic>");
     this->DeclareOption(grp2, "hdri", "", "Path to an image file that will be used as a light source", options.getAsStringRef("render.background.hdri"), LocalHasDefaultNo, MayHaveConfig::YES, "<file path>");
+    this->DeclareOption(grp2, "hdri-hide", "", "When using a HDRI, use it as a light source but hide the background skybox", options.getAsBoolRef("render.background.hdri.hide"), HasDefault::YES, MayHaveConfig::YES);
```

Comment:
If I'm correct, it's the first option we are using to disable something. Why not something like `show-skybox`? Is it because it would change the default behavior for the previous version users?

Reply:
As long as we have a default config with `show-skybox`, I guess we're fine?  
I like our philosophy "disable everything by default and let the user enable what they want".

Reply:
I'm afraid we'll run into difficulties if we want to change the options later.  
But I'll not push further, let's stick with that for now.

Reply:
https://github.com/f3d-app/f3d/issues/927